changelogs/fragments/3393-pkgng-many_packages_one_command.yml

```
@@ -0,0 +1,5 @@
+minor_changes:
+  - pkgng - packages being installed (or upgraded) are acted on in one command (per action)
+    (https://github.com/ansible-collections/community.general/issues/2265)
```

```suggestion
    (https://github.com/ansible-collections/community.general/issues/2265).
```
Out of curiosity: does this also work wiht older versions of the package manager (see the part of #2265 which asks whether this works with older verisons as well)?

---

This has been a feature of pkg(ng) since pre-1.0 (see [commit](https://github.com/freebsd/pkg/commit/0f9e6eef80766ee5649fdd5aad81008cc740bc05)). It just was never supported in the Ansible module.